evap/evaluation/templatetags/evaluation_filters.py

```diff
@@ -21,8 +21,6 @@
     Evaluation.State.PUBLISHED: _("published"),
 }
 
-STR_TO_STATE = {s: i for i, s in Evaluation.STATE_STR_CONVERSION.items()}
-
```

Comment:
I could not find any usages of this dict and removing it did not break any tests, is there a reason to keep it around?

Reply:
I don't think so, probably left unused after some past change, removing should be fine. Git's pickaxe functions (`git log -S STR_TO_STATE`) might show you whether/when the last usage was removed.

Reply:
@niklasmohrin you added this in 2021, but even at that commit (https://github.com/e-valuation/EvaP/tree/37508eb2c426d1e61437e24832a12dbad0d95d12) I do not find any usages

Reply:
:thinking: :thinking: :thinking: 
![image](https://github.com/e-valuation/EvaP/assets/47574893/e78b6844-0cd9-4b62-bc47-29d9156baa97)

Huh, yeah, I guess it's fair to delete it then :D